kinbot/conformers.py

```diff
@@ -441,14 +441,13 @@ def check_conformers(self, wait=0):
                                     err = -1
                             if err == 0:
                                 lowest_totenergy = energy + zpe
-                                if self.species.wellorts:
-                                    base_imag_freq = freq[0]
                         if energy + zpe < lowest_totenergy:
```

Comment:
placement of `base_imag_freq` was causing an error about the variable being called before being assigned below. 

So the line is moved to below so that it can be assigned before being called.